src/update_handler.c

```diff
@@ -1424,12 +1424,12 @@ static gboolean mount_and_run_slot_hook(const gchar *hook_name, const gchar *hoo
 	g_assert_nonnull(hook_name);
 	g_assert_nonnull(hook_cmd);
 
-	/* mount slot */
+	/* try mount slot */
 	g_message("Mounting slot %s", slot->device);
 	res = r_mount_slot(slot, &ierror);
 	if (!res) {
-		g_propagate_error(error, ierror);
-		goto out;
+		g_warning("Ignoring mount error before slot hook error: %s", ierror->message);
```

Comment:
This message doesn't make sense here. I guess it is copied and modified from below?

Please also ensure that this applies to pre-install handlers only as mentioned in the other comment.